plugin-packages/spine/src/spine-component.ts

```
@@ -106,20 +112,29 @@ export class SpineComponent extends RendererComponent {
+      mask: 0,
+      maskMode: MaskMode.NONE,
+    };
     this.item.getHitTestParams = this.getHitTestParams.bind(this);
+
+    const maskProps = (data as MaskProps).mask;
+
+    if (maskProps && maskProps.ref) {
+      maskProps.ref = this.engine.findObject((maskProps.ref as unknown as spec.DataPath));
+    }
+    this.rendererOptions.maskMode = this.maskManager.getMaskMode(data as MaskProps);
+    this.rendererOptions.mask = this.maskManager.getRefValue();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)